google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardDeployTask.java

```diff
@@ -47,6 +49,9 @@ public AppEngineStandardDeployTask(
 
   @Override
   public void execute(ProcessStartListener startListener) {
+    UsageTrackerProvider.getInstance()
```

Comment:
I wonder if there's value in tracking compat vs non-compat deployments. This would have to be added here since compat deployments follow the 'standard' deployment flow.